roles/nova-data/tasks/ssh.yml

```diff
@@ -20,6 +21,12 @@
     group: nova
     mode: 0600
 
+- name: nova known_hosts root symlink (for libvirt)
+  file:
+    src: "{{ nova.state_path }}/.ssh/known_hosts"
+    path: /root/.ssh/known_hosts
+    state: link
```

Comment:
While we could potentially point libvirt directly at nova's known_hosts file, I can't get more than one argument to the live_migration_uri string. Perhaps a bug in Nova or libvirt, but to work around it we're going to use a symlink to the known_hosts that libvirt will default to.